xtc/src/history.rs

```diff
@@ -14,7 +17,7 @@ pub struct HistoryBuffer {
 impl Default for HistoryBuffer {
     fn default() -> Self {
         HistoryBuffer {
-            history: History::<Principal, IcBackend>::new(5_000, 2000),
+            history: History::<Principal, IcBackend>::new(4_300_000, 2000),
```

Comment:
@qti3e are you happy with these values?

Reply:
No, `4_300_00 / 2000 = 2150`, and that's a lot of chunks to write, it'll slow down a lot of transactions since we're still not using `ic_cdk::block_on/ic_kit::spwan`, also since `4_300_000 % 2000 = 0`, it means after each flush the recent data is also flushed.

I'd go with `new(504_000, 5_000)`

It keeps the chunk size less than 2MB, but also finishes the flush in 100 writes.